### ISSUE TITLE
use syntax highlight theme that supports `toml`

### ISSUE THREAD

**Issue by spitfire05** [axodotdev/oranda#236]
## Oranda version

Built from `544e447caf0ecc7d1c7f2d685842755f2538023e` (today's HEAD).

## What is wrong

Using [portproxy](https://github.com/spitfire05/portproxy) to test.

In the README.md file I have a `toml`-annotated code section, that is not being rendered to oranda's html:

```html
<p>Config should contain one or more <code>[[proxy]]</code> elements, that define the port mappings:</p>
<!-- Code block should be here! -->

<h2>Run</h2>
```

However, if I remove the `toml` annotation from markdown, the code block is properly rendered (even with some basic highlighting, it seems).

```html
<p>Config should contain one or more <code>[[proxy]]</code> elements, that define the port mappings:</p>
<pre style="background-color:#263238;"><span style="color:#89ddff;">[[</span><span style="color:#eeffff;">proxy</span><span style="color:#89ddff;">]]
</span><span style="color:#eeffff;">listen </span><span style="color:#89ddff;">= "</span><span style="color:#c3e88d;">localhost:8080</span><span style="color:#89ddff;">"        #</span><span style="color:#eeffff;"> local address to listen on
</span><span style="color:#eeffff;">connect </span><span style="color:#89ddff;">= "</span><span style="color:#c3e88d;">some-server.lan:8485</span><span style="color:#89ddff;">" #</span><span style="color:#eeffff;"> remote </span><span style="color:#89ddff;">(</span><span style="color:#eeffff;">or local</span><span style="color:#89ddff;">)</span><span style="color:#eeffff;"> address to connect to
</span></pre>

<h2>Run</h2>
```

## Expected behavior

Markdown code blocks annotated with `toml` should be properly rendered to output html.

**Comment by ashleygwilliams** [comment]
hey @spitfire05 ! thanks for filing! on a naive glance, i *suspect* this is related to the syntax highlighting theme/library not supporting toml and therefore returning None somewhere in the build, causing the block to just be dropped entirely as opposed to rendered but not highlighted! will take a look in a bit cc @SaraVieira

**Comment by ashleygwilliams** [comment]
by the way- we started a discord if you'd like to join us and chat there!! would love to have you: https://discord.gg/A8QQkCM5

**Comment by spitfire05** [comment]
> hey @spitfire05 ! thanks for filing! on a naive glance, i _suspect_ this is related to the syntax highlighting theme/library not supporting toml and therefore returning None somewhere in the build, causing the block to just be dropped entirely as opposed to rendered but not highlighted! will take a look in a bit cc @SaraVieira

Yeah, that's what I suspect as well, pure guess though, did not have a chance to glance onto the source at all yet.

Also, thanks for the discord invite, just joined 👍

**Comment by ashleygwilliams** [comment]
@spitfire05 so this should be halfway solved with #237, the next step is to support toml blocks

**Comment by manyinsects** [comment]
Merged in #452
